src/utils.py

```diff
@@ -839,7 +839,11 @@ def run_policy(
     Terminates when any of these conditions hold:
     (1) the termination_function returns True
     (2) max_num_steps is reached
-    (3) policy() raises any exception of type in exceptions_to_break_on
+    (3) policy() or step() raise an exception of type in exceptions_to_break_on
```

Comment:
raises*

Reply:
You want this change even though the line will break?

Reply:
no